frontend/src/metabase/setup/components/CompletedStep/CompletedStep.styled.tsx

```diff
@@ -5,25 +5,27 @@ import { color } from "metabase/lib/colors";
 export const StepRoot = styled.section`
   display: flex;
   flex-direction: column;
-  align-items: center;
-  padding: 2rem;
-  margin-bottom: 1.75rem;
+  padding: 2.5rem;
   border: 1px solid ${color("border")};
   border-radius: 0.5rem;
   background-color: ${color("white")};
+  gap: 32px;
+  margin-bottom: 1.75rem;
```

Comment:
Why do we remove `margin-bottom` [on master](https://github.com/metabase/metabase/pull/46783/files#diff-32fa510a1d23f2fe49fd596fc072a20377e836c9d804c3f3b51776719747de2e) but leave it in the backport?

Reply:
That was a bug fixed with https://github.com/metabase/metabase/pull/47021/files which is part of this backport